cartography/cli.py

```diff
@@ -368,15 +368,15 @@ def _build_parser(self):
         )
         return parser
 
-    def main(self, argv):
+    def main(self, argv: str) -> int:
         """
         Entrypoint for the command line interface.
 
         :type argv: string
         :param argv: The parameters supplied to the command line program.
         """
         # TODO support parameter lookup in environment variables if not present on command line
-        config: cartography.config.Config = self.parser.parse_args(argv)
+        config: argparse.Namespace = self.parser.parse_args(argv)
```

Comment:
We do some duck typing with cartography.config.Config so that it has the same fields as the object returned by parse_args, but teeechnically parse_args returns an argparse.Namespace object so I'll type-annotate it properly here.